src/surge-xt/gui/SurgeGUIEditor.cpp

```diff
@@ -2697,11 +2697,15 @@ long SurgeGUIEditor::unapplyParameterOffset(long id) { return id + start_paramta
 void SurgeGUIEditor::toggleMPE()
 {
     this->synth->mpeEnabled = !this->synth->mpeEnabled;
+
     if (statusMPE)
     {
+        this->synth->resetPitchBend(-1);
+
```

Comment:
Do we want to reset if we toggle on also? So always do a pitch bend reset when changing between modes?

Reply:
Yeah I think it's fine in both cases tbh. When toggling on is what tripped Luna up, but it's fine to have it when toggling off too.